spirv_msl.hpp

```diff
@@ -76,8 +76,10 @@ class CompilerMSL : public CompilerGLSL
 	// Options for compiling to Metal Shading Language
 	struct Options
 	{
-		bool enable_point_size_builtin = true;
 		std::string entry_point_name;
+		float msl_version = 1.2f;
```

Comment:
Having a floating point value for version seems a bit dubious to me. Can we make this an integer like HLSL? We could have a static function which creates this integer version ala CompilerMSL::version(major, minor)

Reply:
I've changed this to an int and added setter and validation functions.